music21/key.py

```diff
@@ -996,6 +996,56 @@ def tonicPitchNameWithCase(self):
             tonic = tonic.lower()
         return tonic
 
+    def deriveByKeySignature(self, KeySig: Union[KeySignature, str]) -> List['Key']:
+        '''
+        Derive a fitting Key from the KeySigntature and the tonic of this Key.
+        Thew new key will have the same tonic as this key and the elternated pitches from
+        the key and the key signature. Conflicting alternating keys are overriden by the
```

Comment:
"overridden". also--is "conflicting altered pitches" is more your meaning? I can't quite follow it as written.

Reply:
You're rigth, i did a rewording for this.

>>> If alternated pitches of the same step exist in Key and KeySignature, the alternated pitch is taken from the Keysignature.

Sound better ?

Reply:
> Thanks for finding the issue with `intervalList`! Better to patch it than work around it, so I opened #785.

How do I rebase my commits so that the tests don't stumble over this issue ?